sign-in.txt

```diff
@@ -3,4 +3,9 @@
 //TODO: Please put your name and github username AS SHOWN below ... 
 //       ... then commit and create pull request to develop
 
-Brian Bansenauer : Bansenauer-Cascadia
\ No newline at end of file
+<<<<<<< issue12sruble
+Brian Bansenauer : Bansenauer-Cascadia
+Sim Ruble : sruble  
+=======
+Dr. Brian Bansenauer:Bansenauer-Cascadia
+>>>>>>> develop
```

Comment:
Sim - this line is just showing you where the develop branch code ends. Incorporate any changes from develop into your code to make the file complete and correct (for instance, any names missing from your list) and remove the ">>>>>>> develop" line before you commit